src/documents/ViewAPI.md

```diff
@@ -0,0 +1,174 @@
+to be armed again.
+
+### fluid.deadMansBlur(control, options)
+
+* `control {jQueryable}` The control for which a blur interaction is to be managed.
+* `options {DeadMansBlurOptions}` A set of options managing the interaction, described in [table below](#structure-of-deadmansbluroptions-).
+* Returns: `{Component}` A component managing the interaction
+
+This utility is designed to compensate for the fact that delivery of blur events in browsers has always been extremely unreliable. Sometimes blur events
+are delivered simply because focus has moved to another control within the same interaction region, they are not delivered at all, or in some even more
+perverse cases (especially on Internet Explorer) are delivered *after* the focus event which morally causes them. 
+
+This utility uses the following strategy: on receiving a standard `blur` event on the `control`, a timer is started which waits for a short period of
+time (options.delay, defaults to 150ms) to discover whether the reason for the blur interaction is that either a focus or click is being serviced on a nominated
+set of "exclusions" (`options.exclusions`, a free hash of elements or jQueries, as accepted by `fluid.globalDismissal`).
+
+If no such event is received within the window, `options.handler` will be called with the argument `control`, to service whatever interaction is required of the
+blur. If, further, a click is received *outside* the list of exclusions (as per `fluid.globalDismissal`), `options.handler` will be notified immediately.
+
+Unlike `fluid.globalDismissal`, this interaction is permanent and will persist as long as the nominated DOM elements are in the DOM.
+
+*** NOTE: *** The behaviour of this API has not been vetted for several years against the behaviour of our currently supported browsers, and it also has some deficits which
+require reimplementation (e.g. that its handlers can't be removed from the target elements). It should be considered as an unstable API - however it is still being found
+broadly useful.
+
+#### Structure of `DeadMansBlurOptions`
+ 
+| Name | Type | Description | Default |
+|------|------|-------------|---------|
+|`exclusions`|<code>Object: String  → DomElement&#124;jQuery</code>| A free hash of names onto nodes that a click ***outside*** of will trigger `handler`, *** OR *** that a focus event received within the time window of `delay` from a blur event on `control` will *** cancel *** notification of `handler`| none |
+|`handler`|`Function: (jQueryable)  →  None`| A function which will be invoked when the component has determined that a meaningful blur has been triggered on `control`. It will be invoked with `control` as the argument.| none |
+|`delay`|`Number`| The interval of time that the component will wait after receiving a `blur` event on `control` in order to determine that the reason is that another element within `exclusions` is being focused.|150|
```

Comment:
I know you mentioned above that the delay is milliseconds, but it would probably be a good idea to add that info to this description, too, eg. "The interval of time (in milliseconds) that ..."